packages/api/app/routes.js

```diff
@@ -32,6 +33,10 @@ if (process.env.FAKE_AUTH === 'true') {
   ensureAuthorized = token.ensureAuthorized;
 }
 
+if (process.env.oauthProvider === 'launchpad') {
```

Comment:
If the token has been authorized, it should be validate for 1? hour, and within the hour, cumulus shouldn't have to validate  this token against Launchpad again.  The validation against Launchpad is expensive.

Reply:
actually, I see the token is saved